oc.spec

```diff
@@ -28,51 +35,56 @@
 %global make %{os_git_vars} && make SOURCE_GIT_TAG:="${OS_GIT_VERSION}" SOURCE_GIT_COMMIT:="${OS_GIT_COMMIT}" SOURCE_GIT_MAJOR:="${OS_GIT_MAJOR}" SOURCE_GIT_MINOR:="${OS_GIT_MINOR}" SOURCE_GIT_TREE_STATE:="${OS_GIT_TREE_STATE}"
 %endif
 
-%if ! 0%{?local_build:1}
-Source0:        https://%{import_path}/archive/%{commit}/%{name}-%{version}.tar.gz
-%endif
-Name:           openshift-clients
+Name:           %{package_name}-clients
 Version:        %{version}
 Release:        %{release}%{dist}
 Summary:        OpenShift client binaries
 License:        ASL 2.0
 URL:            https://%{import_path}
 
+%if ! 0%{?local_build}
+Source0:        https://%{import_path}/archive/%{commit}/%{name}-%{version}.tar.gz
+%endif
+
 # If go_arches not defined fall through to implicit golang archs
 %if 0%{?go_arches:1}
 ExclusiveArch:  %{go_arches}
 %else
 ExclusiveArch:  x86_64 aarch64 ppc64le s390x
 %endif
 
-#BuildRequires:  bsdtar
 BuildRequires:  golang >= %{golang_version}
 BuildRequires:  krb5-devel
 BuildRequires:  rsync
 
-Provides:       atomic-openshift-clients
-Obsoletes:      atomic-openshift-clients
+Provides:       atomic-openshift-clients = %{version}
+Obsoletes:      atomic-openshift-clients = %{version}
 Requires:       bash-completion
 
+%if 0%{?ose_build}
+Conflicts:      okd-clients
```

Comment:
be explicit and add this, or leave this out?

Reply:
ref discussion at https://github.com/openshift/origin/pull/23747/files#r321980453

Reply:
`ocp_build`

Reply:
I would want the package names to stay the same `openshift-clients`